packages/pytest-simcore/src/pytest_simcore/helpers/utils_login.py

```diff
@@ -12,9 +12,22 @@
 
 from .utils_assert import assert_status
 
-# WARNING: UserDict is already in https://docs.python.org/3/library/collections.html#collections.UserDict
-# TODO: move this to future simcore_service_webserver.users_models.py
-AUserDict = Dict[str, Any]
+
+# WARNING: DO NOT use UserDict is already in https://docs.python.org/3/library/collections.html#collections.UserDictclass UserRowDict(TypedDict):
+# NOTE: this is modified dict version of packages/postgres-database/src/simcore_postgres_database/models/users.py for testing purposes
+class _UserInfoDictRequired(TypedDict, total=True):
```

Comment:
was confused with the use of the total argument. So True is already the default, and means that all members must be filled. To be honnest I find it a bit unclear, would it not be more clear using the ```Optional``` keywords in the member?

Reply:
``TypedDict`` is not as sophysticated as pydantic and there ``Optional`` is not interpreted as "non-required"  but "can be None".

Actually this trick to split the TypedDict I learned it reading the code of pydantic. SEE https://github.com/samuelcolvin/pydantic/blob/master/pydantic/error_wrappers.py#L16